custom-notification-handler/pom.xml

```diff
@@ -0,0 +1,116 @@
+            </releases>
+        </repository>
+
+        <repository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+
+    </repositories>
+   <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.event.handler.notification</groupId>
+            <artifactId>org.wso2.carbon.identity.event.handler.notification</artifactId>
+            <version>${identity.event.handler.notification.handler.version}</version>
+        </dependency>
+       <dependency>
+           <groupId>org.wso2.carbon.identity.framework</groupId>
+           <artifactId>org.wso2.carbon.identity.event</artifactId>
+           <version>${carbon.identity.framework.version}</version>
+       </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.0</version>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-scr-plugin</artifactId>
+                <version>1.7.2</version>
+                <executions>
+                    <execution>
+                        <id>generate-scr-scrdescriptor</id>
+                        <goals>
+                            <goal>scr</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Private-Package>
+                            org.wso2.carbon.sample.internal.*,
+                        </Private-Package>
+                        <Export-Package>
+                            ! org.wso2.carbon.sample.internal.*,
+                            org.wso2.carbon.sample.*,
+                        </Export-Package>
+                        <Import-Package>
+                            org.wso2.carbon.identity.event.*,
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
It would be better to add imported classes alongside with versions tested in this component, under the  `<Import-Package>` tag and avoid using '<DynamicImport-Package>*</DynamicImport-Package>' unless we have a specific reason to use it, in order to avoid dependency issues.